client/state/analytics/middleware.js

```diff
@@ -1,11 +1,17 @@
+/**
+ * Internal dependencies
+ */
+
 import analytics from 'lib/analytics';
 import has from 'lodash/has';
 import invoke from 'lodash/invoke';
+import { isTracking } from 'state/selectors/is-tracking';
```

Comment:
Since you're [default exporting](https://github.com/Automattic/wp-calypso/pull/12332/files#diff-42017cd4b61c581d406570291f19490dR8) it from that file, you need to drop the curly braces in the `import`, or you'll get

![image](https://cloud.githubusercontent.com/assets/96308/24675825/d7a529e0-1980-11e7-9248-e74f96a2e553.png)

😉